docs/development/ADRs/0017-Toolchain-Configuration.md

```diff
@@ -0,0 +1,101 @@
+---
```

Comment:
I find a good part of this document not useful to read, but also I don't care too much.

Reply:
The usefulness of this document depends heavily on whether one is thinking about changing any of this design without re-covering the same ground.

Is the lack of interest and perceived usefulness due to that you have no desire to do so? Or do you think it is useless under the premise of wanting to change the design. Or does the writing style distract from the actual information?

Reply:
No it's not a lack of interest I just don't feel addressed neither as a user trying to understand how things work (not really the purpose of an ADR sure), but also not as someone trying to understand design decisions (e.g. why would I care about the definition of the term toolchain to mention just one thing).

Reply:
The definition is provided so that it is clear what the design decisions have been taken for. It is meant to be skipped but then double checked before taking misguided action based on something one has read in the document. Same as all other definitions.

If we had clearly defined terms for components of GT4Py this would not be necessary.